filament/backend/include/private/backend/HandleAllocator.h

```diff
@@ -67,7 +68,8 @@ class DebugTag {
 template<size_t P0, size_t P1, size_t P2>
 class HandleAllocator : public DebugTag {
 public:
-    HandleAllocator(const char* name, size_t size, bool disableUseAfterFreeCheck) noexcept;
+    HandleAllocator(const char* name, size_t size,
+            bool disableUseAfterFreeCheck, bool disableHeapHandleTags = false) noexcept;
```

Comment:
nit: why the `= false` default?

Reply:
I added a default because otherwise there was tons of places that needed to be updated; I think it was in tests. I think `false` was the proper default?

Reply:
got it, but maybe need a TODO here too since we have now two different defaults and already a bit confusing.